Tests/IntegrationTests.XamarinAndroid/packages.config

```diff
@@ -2,4 +2,16 @@
 <packages>
   <package id="DotNetCross.Memory.Unsafe" version="0.2.2" targetFramework="monoandroid51" />
   <package id="Fody" version="1.29.4" targetFramework="MonoAndroid44" developmentDependency="true" />
+  <package id="Remotion.Linq" version="2.1.1" targetFramework="MonoAndroid51" />
```

Comment:
This is another `MonoAndroid` level inconsistent with `Realm.XamarinAndroid/packages.config`